joomlarrssb.xml

```diff
@@ -25,7 +25,6 @@
 					<option value="0">JNO</option>
 					<option value="1">JYES</option>
 				</field>
-				<field name="displayGoogle" type="radio" class="btn-group btn-group-yesno" default="1" label="PLG_CONTENT_JOOMLARRSSB_FIELDSET_DISPLAY_GOOGLE_LABEL" description="PLG_CONTENT_JOOMLARRSSB_FIELDSET_DISPLAY_GOOGLE_DESCRIPTION">

```

Comment:
You need to remove the next two lines <option...

Reply:
Thanks Brian!